README.md

```diff
@@ -23,18 +33,20 @@ The current published repository metadata lives in the [repository](/repository/
 
 The current root is published on a GCS bucket located at `https://storage.googleapis.com/sigstore-tuf-root`.
 
+The pre-production root is published ona GCS bucket located at `https://storage.googleapis.com/sigstore-preprod-tuf-root`.
+
+
+## Sigstore Root Keyholders 
+
+| Keyholder        |  TUF Key ID |  Yubikey Material| Term | 
+| ----- |--------- |  --- | ---- |
+| Joshua Locke       |  `75e867ab10e121fdef32094af634707f43ddd79c6bab8ad6c5ab9f03f4ea8c90` | [18158855](https://github.com/sigstore/root-signing/ceremony/2022-07-12/keys/18158855)  | July 2022 -  | 
```

Comment:
```suggestion
| Joshua Lock       |  `75e867ab10e121fdef32094af634707f43ddd79c6bab8ad6c5ab9f03f4ea8c90` | [18158855](https://github.com/sigstore/root-signing/ceremony/2022-07-12/keys/18158855)  | July 2022 -  | 
```
no e :-)

Reply:
Ah thank you! I don't know why I did that.